example/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,18 +15,20 @@
       <meta-data
         android:name="com.google.android.gms.wallet.api.enabled"
         android:value="true" />
+      <activity android:name=".LaunchActivity" android:exported="true">
+        <intent-filter>
+          <action android:name="android.intent.action.MAIN" />
+          <category android:name="android.intent.category.LAUNCHER" />
+        </intent-filter>
+      </activity>
+
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
         android:launchMode="singleTask"
```

Comment:
Replacing `singleTask` by `singleTop` may improve the app compatibility with deep linking.

Opening a deep link with that `launchMode` will cause the app to be restarted and may cause the lost of the current state.

Reply:
singleTask is what is generally recommended (and the only thing that seems to work well in practice) by Branch.io for instance. As long as the activity that handles deeplinks is singleTask, but the one that handles the launcher intent is not, both the Stripe use case, and the Branch use case seem to work well in the same app at the same time.